packages/cactus-plugin-ledger-connector-xdai/src/main/typescript/plugin-ledger-connector-xdai.ts

```diff
@@ -158,6 +166,36 @@ export class PluginLedgerConnectorXdai
 
   async registerWebServices(app: Express): Promise<IWebServiceEndpoint[]> {
     const webServices = await this.getOrCreateWebServices();
+    app.use(
+      OpenApiValidator.middleware({
+        apiSpec: this.getOpenApiSpecs(),
+        validateApiSpec: false,
+      }),
+    );
+    app.use(
```

Comment:
@elenaizaguirre Can you move this to the API server so that it hooks it up at plugin registration time? That way the plugins don't have to depend on the openapi validator package at all and we will avoid issues where different versions of it in different plugins are conflicting.